[2024-01-11 11:32:11,957] [INFO] [test_01_workspace_mnt.py] [line 44] 插件管理-根据自定义插件ID获取场景数据-错误的id请求路径为:/OneHit/plugin/scene/87987,请求体为:{}
[2024-01-11 11:32:11,976] [INFO] [test_01_workspace_mnt.py] [line 77] 插件管理-根据自定义插件ID获取场景数据-错误的id状态码：404，响应结果:{"code":"RESOURCE_NOT_FOUND","message":"未找到指定插件"}
